bdk_core/src/tx_graph.rs

```diff
@@ -51,8 +51,8 @@ impl TxGraph {
     }
 
     /// Returns true when graph contains given tx of txid (whether it be partial or full).
-    pub fn contains_txid(&self, txid: Txid) -> bool {
-        self.txs.contains_key(&txid)
+    pub fn contains_txid(&self, txid: &Txid) -> bool {
```

Comment:
why change this?

Reply:
Because you only need a ref in the `hashMap::contains_key()`?? This was causing a move error somewhere downstream in the electrum example.. And required an extra cone, while the cloned object is dropped at this location only.. That was my reasoning behind this.. Open to change if this causes some other problems..